src/atoms/Layout/index.js

```diff
@@ -0,0 +1,81 @@
+import React, { PropTypes } from 'react';
+import classnames from 'classnames';
+
+import styles from './layout.module.scss';
+
+function Layout ( props ) {
+  const {
+    children,
+    childCols,
+    breakpoint,
+    horizontalAlign,
+    verticalAlign
+  } = props;
+
+  let colsIdx = -1;
+
+  const kids = React.Children.map( children, ( child, idx ) => {
+    const length = !children.length ? 1 : children.length;
+    const childColsLength = childCols.length;
+
+    colsIdx = childColsLength == 1 || ( (idx + 1) > childColsLength && (idx + 1) % childColsLength === 1 ) ? -1 : colsIdx;
```

Comment:
This logic is hard to decipher. I'm imagining that it's supposed to loop over the ratios if you have more children than ratios provided? I personally think that something like this should have a little explanatory comment, but general PG policy is that if it needs a comment, you should refactor it to make it easier to understand.